### PR TITLE
[Bing search] Update SDK and lg

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/experimental/bingsearchskill/Services/BotServices.cs [hunk 1]
```diff
@@ -48,20 +48,20 @@ public BotServices(BotSettings settings, IBotTelemetryClient client)
...
-                        {
-                            KnowledgeBaseId = kb.KbId,
-                            EndpointKey = kb.EndpointKey,
-                            Host = kb.Hostname,
-                        };
-                        var qnaMaker = new QnAMaker(qnaEndpoint);
-                        set.QnAServices.Add(kb.Id, qnaMaker);
-                    }
-                }
+                //if (config.Knowledgebases != null)
```
Comment (lzc850612) [inline skills/csharp/experimental/bingsearchskill/Services/BotServices.cs]: the latest fix should be

foreach (var kb in config.Knowledgebases)
                {
                    var qnaEndpoint = new QnAMakerEndpoint()
                    {
                        KnowledgeBaseId = kb.KbId,
                        EndpointKey = kb.EndpointKey,
                        Host = kb.Hostname,
                    };

                    set.QnAConfiguration.Add(kb.Id, qnaEndpoint);
                }

Comment (lzc850612) [reply skills/csharp/experimental/bingsearchskill/Services/BotServices.cs]: i think i mistakenly commented out this part in my email and todo skill PR. can you check all the skills to make sure this is handled properly?

Comment (xieofxie) [reply skills/csharp/experimental/bingsearchskill/Services/BotServices.cs]: haha, sad story..

Comment (KayMKM) [reply skills/csharp/experimental/bingsearchskill/Services/BotServices.cs]: sure. this skill has been fixed.

File: skills/csharp/experimental/bingsearchskill/Utilities/EngineWrapper.cs [hunk 2]
```diff
@@ -57,12 +53,12 @@ public static Activity GetCardResponse(this LocaleTemplateEngineManager manager,
             }
         }
 
-        public static Activity GetCardResponse(this LocaleTemplateEngineManager manager, string templateId, Card card, StringDictionary tokens = null, string containerName = null, IEnumerable<Card> containerItems = null)
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, string templateId, Card card, IDictionary<string, object> tokens = null, string containerName = null, IEnumerable<Card> containerItems = null)
```
Comment (bobokids) [inline skills/csharp/experimental/bingsearchskill/Utilities/EngineWrapper.cs]: could you refer to the change in wrapper in ITSM skill? change GetResponse to GenerateActivity

Comment (KayMKM) [reply skills/csharp/experimental/bingsearchskill/Utilities/EngineWrapper.cs]: fixed

**Pull request merged**